cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/query/AbstractIndexTestBase.java

```diff
@@ -43,16 +43,16 @@ public void setUp() throws Exception {
         assertThat(ds, is(notNullValue()));
         im = new IndexManager(ds);
         assertThat(im, is(notNullValue()));
-        assertThat(im.getDatabaseQueue(), is(notNullValue()));
+        assertThat(TestUtils.getDBQueue(im), is(notNullValue()));
```

Comment:
It may be preferable to declare
`protected SQLDatabaseQueue queue;`
and set the value in the `setUp` using:
`queue = TestUtils.getDBQueue(im);`
That way the reflection only needs to be performed once per test.